.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,6 +17,8 @@ Here are some guidelines to help the review process go smoothly.
    - Each user facing function should be added to the `cmake-format.json` document
     - Run `cmake-genparsers -f json` on the `.cmake` file as a starting point
    - Each function first line should be `list(APPEND CMAKE_MESSAGE_CONTEXT "rapids.<component>.<function>")`
+   - Internal variables for things like `cmake_parse_arguments` should use `_RAPIDS`
```

Comment:
Is it worth recommending a specific alternative rather than simply proscribing the use of `RAPIDS`? Personally I have been using more specific names in each PR that I make (e.g. `RAPIDS_CYTHON` or `CUCO`).

Reply:
Unless there's a reason that we actually _want_ variables from higher scopes to be visible in lower scopes using a common naming scheme?

Reply:
When I started with `RAPIDS` I did it as I felt that this prefix would be unused by any consumer of rapids-cmake. Therefore we wouldn't accidently de-reference a variable from the calling project. Issue #200 showed that other projects use variables that start with `RAPIDS` and therefore we moved to `_RAPIDS`.

I have the same concern with project specific prefixes like `CUCO`. The calling project could set variables like `CUCO_VERSION` that could influence rapids-cmake internal behavior.

Reply:
I see, yeah that's true. Maybe we should suggest following the same naming convention as the function names (e.g. `rapids_cpm_find` -> `RAPIDS_CPM_FIND_...`)? It obviously gets very long, though, so there's a significant tradeoff. I'll stick to `_RAPIDS` for now and if we ever run into internal conflicts I guess we can worry about this then. Worst case it's another find-and-replace job.